module/icp/include/sys/ia32/asm_linkage.h

```diff
@@ -30,9 +30,28 @@
 #include <sys/stack.h>
 #include <sys/trap.h>
 
-#if defined(__linux__) && defined(CONFIG_SLS)
-#define	RET	ret; int3
-#else
+#if defined(_KERNEL) && defined(__linux__)
+#include <asm/linkage.h>
+#endif
+
+#if !defined(ENDBR)
+#if defined(__ELF__) && defined(__CET__) && defined(__has_include)
+#if __has_include(<cet.h>)
```

Comment:
There is a check style complaint here.

Reply:
Yes, but it's a false-positive. It treats <> in `__has_include(<cet.h>)` as comparison operators, as I would write for example `if (a<b)`.

@behlendorf, do I need to mark this place in the code with some checkstyle-ignore tag?

Reply:
Exactly, you can add a CSTYLED comment above the problematic line to silence the false positive.

```
/* CSTYLED */
#if __has_include(<cet.h>)
```

You can also run `make checkstyle` locally before repushing to make sure the style checker is happy.